### PR TITLE
[SOAR-9700] Threatminer SNYK fix

### PR DESCRIPTION
**Pull request by hpwilson-r7**
### Description

Describe the proposed changes:

  - Change requests version to 2.20.0

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

**Review by TKuzas-R7 — APPROVED** [review]

File: plugins/threatminer/help.md [hunk 1]
```diff
@@ -13,66 +13,98 @@ email addresses. In addition, it can assist in malicious attachment detection wh
 
  _This plugin does not contain any requirements._
 
+# Supported Product Versions
+
+* 2021-08-17
```
Comment (ablakley-r7) [inline plugins/threatminer/help.md:18]: Is this the newest date we have tested the plugin?

File: plugins/threatminer/help.md [hunk 2]
```diff
@@ -13,66 +13,98 @@ email addresses. In addition, it can assist in malicious attachment detection wh
...
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|response|response|False|Response|
+|Name|Type|Required|Description|Example|
+|----|----|--------|-----------|-------|
+|response|response|False|Response|{'response': {'status_code': '404', 'status_message': 'No results found.', 'results': []}}|
 
+Example output:
```
Comment (ablakley-r7) [inline plugins/threatminer/help.md:54]: Can an example output be added?

File: plugins/threatminer/help.md [hunk 3]
```diff
@@ -13,66 +13,98 @@ email addresses. In addition, it can assist in malicious attachment detection wh
...
+_This plugin does not contain a connection._
 
 ## Technical Details
 
 ### Actions
 
 #### AV Report
 
-This action is used to fetch information related to a domain by URIs, certificates, or related samples.
+This action is used to AV Report.
```
Comment (ablakley-r7) [inline plugins/threatminer/help.md:32]:
```suggestion
This action is used to obtain an AV Report.
```

File: plugins/threatminer/help.md [hunk 4]
```diff
@@ -13,66 +13,98 @@ email addresses. In addition, it can assist in malicious attachment detection wh
...
+{
+  "query": "Trojan.Enfal"
+}
+```
 
+##### Output
+
+|Name|Type|Required|Description|Example|
+|----|----|--------|-----------|-------|
+|response|response|False|Response|{'response': {'status_code': '404', 'status_message': 'No results found.', 'results': []}}|
```
Comment (ablakley-r7) [inline plugins/threatminer/help.md:81]: Can we add an example output?

File: plugins/threatminer/help.md [hunk 5]
```diff
@@ -13,66 +13,98 @@ email addresses. In addition, it can assist in malicious attachment detection wh
...
+```
 
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|response|response|False|Response|
+|Name|Type|Required|Description|Example|
+|----|----|--------|-----------|-------|
+|response|response|False|Response|{'response': {'status_code': '404', 'status_message': 'No results found.', 'results': []}}|
```
Comment (ablakley-r7) [inline plugins/threatminer/help.md:107]: Can we add an example output?

File: plugins/threatminer/help.md [hunk 6]
```diff
@@ -81,16 +113,25 @@ This action fetches information related to a domain by URIs, certificates, or re
...
+```
 
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|response|response|False|Response|
+|Name|Type|Required|Description|Example|
+|----|----|--------|-----------|--------|
+|response|response|False|Response|{'response': {'status_code': '404', 'status_message': 'No results found.', 'results': []}}|
```
Comment (ablakley-r7) [inline plugins/threatminer/help.md:134]: Can we add an example output?

File: plugins/threatminer/help.md [hunk 7]
```diff
@@ -99,102 +140,147 @@ This action fetches information related to an email address.
...
+{
+  "email": "user@example.com"
+}
+```
 
+##### Output
+
+|Name|Type|Required|Description|Example|
+|----|----|--------|-----------|-------|
+|response|response|False|Response|{'response': {'status_code': '404', 'status_message': 'No results found.', 'results': []}}|
```
Comment (ablakley-r7) [inline plugins/threatminer/help.md:159]: Same for all here and below.

**Review by ablakley-r7 — CHANGES_REQUESTED** [review]

File: plugins/threatminer/plugin.spec.yaml [hunk 8]
```diff
@@ -50,14 +51,16 @@ actions:
...
+        description: Virus name to query
         required: true
+        example: Trojan.Enfal
     output:
       response:
         title: Response
         type: response
         required: false
         description: Response
+        example: { "response": { "status_code": "404", "status_message": "No results found.", "results": [ ] } }
```
Comment (ablakley-r7) [inline plugins/threatminer/plugin.spec.yaml:63]: I think these examples need changed to `"status_code": 404`

**Review by ablakley-r7 — CHANGES_REQUESTED** [review]

**Review by ablakley-r7 — APPROVED** [review]

**Review by dlaverty-r7 — APPROVED** [review]

**Pull request merged**